packages/components/src/toolbar/toolbar/types.ts

```diff
@@ -22,4 +22,11 @@ export type ToolbarProps = {
 	 * @default undefined
 	 */
 	variant?: 'unstyled' | undefined;
+	/**
+	 * Orientation of the toolbar keyboard interactions.
+	 * horizontal or vertical
```

Comment:
```suggestion
```

I think we can omit this because the docgen will show the available enums from the type data.